pkg/storage/s3/checker.go

```diff
@@ -26,22 +26,29 @@ func (s *Storage) Exists(ctx context.Context, module, version string) (bool, err
 		Bucket: aws.String(s.bucket),
 		Prefix: aws.String(fmt.Sprintf("%s/@v", module)),
 	}
+	var count int
+	err := s.s3API.ListObjectsPagesWithContext(ctx, lsParams, func(loo *s3.ListObjectsOutput, lastpage bool) bool {
```

Comment:
Does this commit not fix this: https://github.com/gomods/athens/commit/92150d050063ae014bee54486afc9d41fa38298e